Tensile/KernelWriterAssembly.py

```diff
@@ -5430,6 +5417,62 @@ def lraFinalOffset(self, kernel, tP):
 
     return kStr
 
+  ##############################################################################
+  # Local Read Addresses offset conversion for DTL + NLC > 1
+  ##############################################################################
+  def lraOffsetConversionForDTLandNLC(self, kernel, tP, offset_val, generateAsm=False, \
```

Comment:
Any way we can refactor this to keep the code modularized in the component? Might be good to avoid moving code out of components back into KernelWriterAssembly

Reply:
Same logic needs to be applied to 2 different locations.
Previously, they were implemented separately and I tended to forget to fix one location.
Gathering same logic in 1 function is a good way to maintain consistency of code.
I spent time to came up with this implementation and this was the best I could do.
Do you have any better idea for this implementation?

Reply:
Not particularly, mostly just wondering if it's possible / makes sense to have the function in the LocalRead component rather than KernelWriterAssembly.